src/azure-account.ts

```diff
@@ -387,10 +387,23 @@ export class AzureLoginHelper {
 						ignoreFocusOut: true
 					});
 					if (armUrl) {
-						await config.update(customCloudArmUrlKey, armUrl, getCurrentTarget(config.inspect(customCloudArmUrlKey)));
+						const tenantId = await window.showInputBox({
+							prompt: localize('azure-account.enterTenantId', "Enter the Tenant Id"),
+							placeHolder: "Tenant Id..",
+							ignoreFocusOut: true
+						});
+						if (tenantId) {
```

Comment:
Will the user always have a specific tenant id when configuring a custom cloud? Or would there be times when using "common" is ok? If "common" is an option, maybe the placeholder text should give it as an example

Reply:
Yeah, as far as my concern, I believe each Custom Cloud will have each specific tenant id, and I don't think common is an option we can use for Custom Cloud. @bganapa Please correct me if I am wrong.

Reply:
And also Azure Custom Cloud would have different tenant id for AAD and ADFS, therefore, it isn't that easy to just give a simple placeholder to represent both.

Reply:
Yes, common is never an option for azure stack, it will always have a specific tenant id in case of AAD. in ADFS case, its just 'adfs'